app/templates/exchange/create.pug

```diff
@@ -27,10 +27,9 @@
               span.blue.pointer.hidden-sm.hidden-md.hidden-lg(translate="EXCHANGE.CHANGE_EMAIL" ng-click="$ctrl.view('email')" ng-show="accountForm.email.$error.registered && $ctrl.inMobileBuy")
             input(ng-model="$ctrl.state.email" name="email" disabled)
           div(ng-show="accountForm.email.$error.registered")
-            .help.mb-10
-              span.flex-row(ng-click="$ctrl.view('email')")
+            .help
+              div.flex-column(ng-click="$ctrl.view('email')")
```

Comment:
The 'change email' link was breaking onto the next line - I thought this was a better presentation.